src/cli/run.ts

```diff
@@ -129,6 +135,17 @@ export const runTask = async (
   sys?: d.CompilerSystem
 ): Promise<void> => {
   const logger = config.logger ?? createLogger();
+
+  let devMode = config.devMode ?? DEFAULT_DEV_MODE;
+  // default devMode false
+  if (config.flags?.prod) {
+    devMode = false;
+  } else if (config.flags?.dev) {
+    devMode = true;
+  } else if (!isBoolean(config.devMode)) {
+    devMode = DEFAULT_DEV_MODE;
+  }
```

Comment:
this snippet is copypasta from a bit in `validate-config.ts` - basically just sets a default `devMode` value. We could pull this out into a little helper if we wanted.